engine/table/src/main/java/io/deephaven/engine/table/impl/TableDefaults.java

```diff
@@ -240,59 +238,101 @@ default Table formatColumns(String... columnFormats) {
         return updateView(selectColumns);
     }
 
-    default void validateColor(String color) throws IllegalArgumentException {
-        new Color(color); // Throws if it can't create it
-    }
-
-    default void validateDataBarOptions(String column, String valueColumn, AxisOptions axis, Double min, Double max,
-            String positiveColor, String negativeColor, ValuePlacementOptions valuePlacement,
-            DirectionOptions direction, Double opacity, String markerColumn, String markerColor)
+    private void validateDataBarOptions(String column, String valueColumn, DataBarAxisOptions axis, double min,
```

Comment:
Yeah, definitely should check how this behaves with other column types, like BigInteger, BigDecimal, int, long, etc.

Reply:
Verified that these work. I converted them into the double type using the coerceToDouble function we have. We loss some precision in the conversion but I don't think it will be noticeable on the UI side for differences that small